portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,19 +57,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   /**
-   * Converts an ArrayList<String> into a JSON string using manual String concatentation.
+   * Converts an ArrayList<String> into a JSON string.
    */
-  private String convertToJson(ArrayList<String> facts) {
+  private String convertToJson(ArrayList<String> comments) {
     String json = "{";
```

Comment:
The variable name |json| could be a little more informative. What is this JSON representing? All the comments submitted so far to the web app?

Reply:
Yeah, I took that from provided code elsewhere where they just used "json"